src/components/common/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,79 @@
+import clsx from 'clsx';
+
+import PaginationLeftArrow from '#/icons/icon-pagination-left-arrow.svg';
+import PaginationRightArrow from '#/icons/icon-pagination-right-arrow.svg';
+
+import styles from './Pagination.module.css';
+
+interface PaginationProps {
+  pageNumber: number;
+  totalPages: number;
+  handlePaginationByClick: (value: number) => void;
+}
+
+/**
+ * @param pageNumber 현재 페이지 넘버
+ * @param totalPages 모든 페이지 수
+ * @param handlePaginationByClick 숫자를 인자로 받는 void 함수
+ */
+function Pagination({ pageNumber, totalPages, handlePaginationByClick }: PaginationProps) {
+  const pageArr = [];
+  for (let i = 1; i <= totalPages; i++) pageArr.push(i);
+
+  const currentPageIndex = Math.ceil(pageNumber / 5);
+  const showPages = pageArr.slice((currentPageIndex - 1) * 5, currentPageIndex * 5);
+
+  return (
+    <div className={styles.container}>
+      <button
+        className={
+          pageNumber === 1
+            ? clsx(styles.paginationNumberWrapper, styles.paginationDisabled)
+            : clsx(styles.paginationNumberWrapper, styles.paginationEnabled)
+        }
```

Comment:
```suggestion
        className={clsx(styles.paginationNumberWrapper,
          pageNumber === 1 ? styles.paginationDisabled : styles.paginationEnabled,
        )}
```
styles.paginationNumberWrapper 한번만 작성할 수 있습니다. 
아래 비슷한 경우가 많은데 참고해주세요.